<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifferenceDialogForm</class>
 <widget class="QDialog" name="DifferenceDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Difference</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QWidget { background: white }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_datasourceTypeTitleLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Applying the difference between two layers.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="m_imgLabel">
        <property name="minimumSize">
         <size>
          <width>112</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Imagem</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="m_inputLayerGroupBox">
     <property name="title">
      <string>Input Layer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="m_inputLayerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="m_inputOnlySelectedCheckBox">
        <property name="text">
         <string>Only selected objects</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="m_differenceLayerGroupBox">
     <property name="title">
      <string>Difference Layer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QComboBox" name="m_differenceLayerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="m_diffOnlySelectedCheckBox">
        <property name="text">
         <string>Only selected objects</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="m_attributesGroupBox">
     <property name="title">
      <string>Attributes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Repository:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Layer Name:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLineEdit" name="m_repositoryLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="m_targetFileToolButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QToolButton" name="m_targetDatasourceToolButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>BD</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="m_newLayerNameLineEdit">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
           <property name="text">
            <string>Help</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="m_okPushButton">
           <property name="text">
            <string>&amp;Ok</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="m_cancelPushButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_inputLayerComboBox</tabstop>
  <tabstop>m_helpPushButton</tabstop>
  <tabstop>m_okPushButton</tabstop>
  <tabstop>m_cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
